hz: 30
max_laser_range: 3.0
offset_pixel: 3
laser_topic_name: "/scan"
map_topic_name: "/map"
laser_frame: "laser"
map_fram: "map"

